src/bin/pg_autoctl/pgsql.c

```diff
@@ -1253,21 +1253,31 @@ parsePgsrSyncStateAndWAL(void *ctx, PGresult *result)
 
 		case 1:
 		{
-			char *strWalLag = PQgetvalue(result, 0, 1);
```

Comment:
We did 1 then 0 because we have to declare variables before any piece of other code in C. Now that both parts are done the same way, I think it's fair to re-order them in their natural order, 0 then 1.

Reply:
done